app/cdap/api/pipeline.js

```diff
@@ -65,6 +65,7 @@ export const MyPipelineApi = {
   getExtensions: apiCreator(dataSrc, 'GET', 'REQUEST', extensionsPath),
 
   getDrafts: apiCreator(dataSrc, 'GET', 'REQUEST', `${pipelineV1AppContextPath}/drafts`),
```

Comment:
Just verified, we could add the query param `?includeConfig=true` to the end of this API: `${pipelineV1AppContextPath}/drafts?includeConfig=true`

https://github.com/cdapio/cdap/blob/develop/cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/DraftHandler.java#L81

Reply:
Do you think it's better to preload all the configs, or get the config when you try to export a draft?

Reply:
It's a great question. It depends on the size of the total configs. If it's too big, it might affect the list loading time. I'm fine with both ways

Reply:
Actually right now for deployed pipelines, it gets config on export. So should keep the same pattern